docs/en_US/Compressor/Overview.md

```diff
@@ -257,7 +273,46 @@ class YourQuantizer(nni.compression.tensorflow.Quantizer):
         """
         pass
 ```
+#### customize backward function
+Sometimes it's necessary for a quantization operation to have a customized backward function, such as Straight-Through Estimator,
```

Comment:
please explain `Straight-Through Estimator`, or add reference link for it.

Reply:
I add a stackoverflow link to shortly explain this concept. There is also a paper reference link in this answer.